tests/common.go

```diff
@@ -3032,11 +2945,15 @@ func CreateCloudCredentialNonAdminUser(provider, name string, uid, orgID string,
 				},
 			},
 		}
-
-		_, err := backupDriver.CreateCloudCredential(ctx, credCreateRequest)
-		if err != nil && strings.Contains(err.Error(), "already exists") {
+	default:
+		return fmt.Errorf("provider [%s] not supported for creating cloud credential", provider)
+	}
+	_, err := Inst().Backup.CreateCloudCredential(ctx, credCreateRequest)
+	if err != nil {
+		if strings.Contains(err.Error(), "already exists") {
 			return nil
 		}
+		log.Errorf("Failed to create cloud credential with name [%s] in org [%s] with [%s] as provider", credName, orgID, provider)
```

Comment:
error log strings should not be capitalized, right?

Reply:
My IDE did not raise any warnings. Capitalized it to distinguish it from direct logging and logging of returned errors.

Reply:
Yes. Make it lower case

Reply:
Done. Thanks @tthurlapati-px.